vendor/bat-native-ads/src/bat/ads/internal/ad_diagnostics/ad_diagnostics_entry.cc

```diff
@@ -0,0 +1,57 @@
+/* Copyright 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/ad_diagnostics/ad_diagnostics_entry.h"
+
+#include <utility>
+
+#include "base/strings/string_number_conversions.h"
+#include "base/values.h"
+
+namespace ads {
+
+namespace {
+
+const char kDiagnosticsEntryName[] = "name";
+const char kDiagnosticsEntryValue[] = "value";
+
+}  // namespace
+
+const char kDiagnosticsAdsEnabled[] = "Ads enabled";
+const char kDiagnosticsLocale[] = "Locale";
+const char kDiagnosticsCatalogId[] = "Catalog ID";
+const char kDiagnosticsCatalogLastUpdated[] = "Catalog last updated";
+const char kDiagnosticsLastUnIdleTimestamp[] = "Last unidle timestamp";
```

Comment:
Usually we want strings like these to be in a .grd file like `brave_components_strings.grd` or they won't be translated. Not sure if that would be relevant for this particular issue, but I think it probably is unless these aren't user-visible.

Reply:
Ah yes these are only for internal use by brave so preferably we keep in English. Some users will try to decipher their meanings but primarily it is for our support teams. Your thoughts?

Reply:
Sure, that's fine by me.